module/extension/FileWatcher/src/FileWatcher.cpp

```diff
@@ -236,6 +236,7 @@ namespace module::extension {
                 if (paths.find(dir) == paths.end()) {
                     auto& p        = paths[path];
                     p.handle       = std::make_unique<uv_fs_event_t>();
+                    uv_fs_event_init(loop.get(), p.handle);
```

Comment:
Is there a delete that needs to be run that should be attached to the deleter?

Reply:
Its freed here https://github.com/NUbots/NUbots/blob/main/module/extension/FileWatcher/src/FileWatcher.cpp#L297

Reply:
I could add it to the deleter.